config/integrations.yml

```diff
@@ -0,0 +1,6 @@
+intercom:
+  client_id: <%= ENV['INTERCOM_APP_KEY'] %>
+  client_secret: <%= ENV['INTERCOM_APP_SECRET'] %>
```

Comment:
Let's use the same names for these. `INTERCOM_APP_KEY = INTERCOM_KEY`  so we can match this with github below.